tests/integration/test_vulnerability_detector/test_scan_results/test_duplicate_feeds.py

```diff
@@ -0,0 +1,183 @@
+        provider_name (str): Provider name of the downloaded feed.
+        provider_json_name (str): Provider json name of the downloaded feed.
+        timeout (int): Timeout of the event.
+
+    Returns:
+        int: number of vulnerabilities inserted
+    """
+    if 'Red Hat Enterprise Linux 8' in provider_name:
+        evm.check_provider_database_update_finish_log(wazuh_log_monitor=wazuh_log_monitor,
+                                                      provider_name=provider_name)
+        evm.check_provider_database_update_finish_log(wazuh_log_monitor=wazuh_log_monitor,
+                                                      provider_name=provider_json_name)
+    else:
+        evm.check_provider_database_update_finish_log(wazuh_log_monitor=wazuh_log_monitor,
+                                                      provider_name=provider_name)
+
+    return cve_db.get_provider_feeds_number()
+
+
+@pytest.mark.parametrize('configuration, metadata', zip(configurations, configuration_metadata), ids=test_case_ids)
+def test_duplicate_feeds(configuration, metadata, set_wazuh_configuration, truncate_monitored_files,
+                         clean_cve_tables_func, setup_log_monitor, restart_modulesd_function):
+    '''
+    description: Check that the vulnerabilities are not repeated in the database when it is updated. To do this, it
+                 updates the database with a custom feed and wait to <update_interval> time to download again the feed
+                 and update the database.
+
+    wazuh_min_version: 4.3.0
+
+    parameters:
+        - configuration:
+            type: dict
+            brief: Wazuh configuration data. Needed for set_wazuh_configuration fixture.
+        - metadata:
+            type: dict
+            brief: Wazuh configuration metadata
+        - set_wazuh_configuration:
+            type: fixture
+            brief: Set the wazuh configuration according to the configuration data.
+        - truncate_monitored_files:
+            type: fixture
+            brief: Truncate all the log files and json alerts files before and after the test execution.
+        - clean_cve_tables_func:
+            type: fixture
+            brief: Clean all the vulnerabilities tables before and after running the test.
+        - setup_log_monitor
+            type: fixture
+            brief: Create the log monitor.
+        - restart_modulesd_function:
+            type: fixture
+            brief: Restart the wazuh-modulesd daemon.
+
+    assertions:
+        - Verify that the number of vulnerabilities inserted in the VULNERABILITIES table of CVE DB is not duplicated.
+
+    input_description:
+        - The `case_duplicate_feeds.yaml` file provides the module configuration for this test.
+
+    expected_output:
+        - 'The update of the <provider_name> feed finished successfully'
+    '''
+    log_monitor = setup_log_monitor
+
+    number_feeds_first = check_update_finish_logs(wazuh_log_monitor=log_monitor,
```

Comment:
```suggestion
    before_feeds_number = check_update_finish_logs(wazuh_log_monitor=log_monitor,
```

Reply:
Done in https://github.com/wazuh/wazuh-qa/pull/2575/commits/73dd596fd88d841d166213eb00ccae04e87dc800